src/installer.ts

```diff
@@ -81,7 +83,10 @@ export async function getNode(versionSpec: string) {
   core.addPath(toolPath);
 }
 
-async function queryLatestMatch(versionSpec: string): Promise<string> {
+async function queryLatestMatch(
```

Comment:
osArch doesn't need to be added as a param here

Reply:
Why is that? It looks like it's used in the switch case immediately below it.

Reply:
That uses `osPlat`

Reply:
Isn't this a use of `os.arch()` that I replaced with `osArch` the parameter?
https://github.com/actions/setup-node/pull/42/files#diff-8c84d074e67a54fc5a18692e0b853feaR159

Reply:
That's a different function (aquireNode vs queryLatestMatch)

Reply:
Oops, my apologies. Line 94 in the switch case of queryLatestMatch:
https://github.com/actions/setup-node/pull/42/files#diff-8c84d074e67a54fc5a18692e0b853feaR94